content/en/docs/gke/deploy/management-setup.md

```diff
@@ -37,8 +37,19 @@ For a more detailed explanation of the changes affecting Kubeflow 1.1 on Google
 
 1. Install [Kustomize v3.2.1](https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv3.2.1).
 
-    Note, Kubeflow is not compatible with later versions of Kustomize. Read [this GitHub issue](https://github.com/kubeflow/manifests/issues/538) for the latest status.
+    **Note:** Kubeflow is not compatible with Kustomize versions above 3.2.1. Read [this GitHub issue](https://github.com/kubeflow/manifests/issues/538) for the latest status.
 
+1. Next, you'll be installing [yq](https://github.com/mikefarah/yq):
+
+    1. Check that you have [Go](https://golang.org) installed by running the `go version`
+       command. If you don't, you can download a binary from
+       [yq's GitHub releases](https://github.com/mikefarah/yq/releases).
+    2. Install yq by running the following command line:
```

Comment:
The current structure makes it sound like step 2 is required even if user downloaded yq binary.

Can we make that clearer?

Reply:
did something, does this work?